modules/sdk/project-templates-panel-app/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -1,5 +1,5 @@
 <%@ include file="/init.jsp" %>
 
 <p>
-	<b><liferay-ui:message key="_portlet_fqn_.caption" /></b>
+	<b><liferay-ui:message key="${artifactId}.caption" /></b>
```

Comment:
I just saw all the other templates having `${artifactId}`, so I made them consistent. But I'll add a test!